test/lib/ls.js

```diff
@@ -2476,6 +2476,12 @@ t.test('ls --json', (t) => {
         },
         'should output json contaning only occurences of filtered by package'
       )
+      t.equal(
+        process.exitCode,
+        0,
+        'should exit with error code 1'
+      )
+      process.exitCode = 0
```

Comment:
Better not to set the exitCode in tests, since that can hide a test failure.  (For example, if the assertion fails, we _want_ the exitCode to stay non-zero!)

Reply:
Oh yeah thats a good point - it makes sense to do in the inverse test because the error code is set to error, but here you're right it could suppress other errors.

I also should update the text to read `should exit with error code 0` 🤦

I'll do both of those now :)